### ISSUE TITLE
GitHub versions not receiving notifications

### ISSUE THREAD

**Issue by DHYTFG** [NekoX-Dev/NekoX#834]
Hello. I'm not getting messages on the GitHub version. I think this is due to the app not running in the background for some reason. With the version from f-droid, there are no such problems. How can I fix this?

**Comment by arielherself** [comment]
You can turn on "Keep-Alive Service" and "Background Connection" in "Notifications and Sounds" settings page to resolve this problem.

**Comment by 0Karakurt0** [comment]
No, I have enabled Background connection, system does not limit Necogram in any way, and yet I see that it has 0 minutes of background power consumption. And don't see notifications therefore

**Comment by NikAr15** [comment]
A few years late but did you find a fix?
